eras/conway/impl/src/Cardano/Ledger/Conway/Rules/GovCert.hs

```diff
@@ -207,7 +213,11 @@ conwayGovCertTransition = do
               Map.adjust
                 ( \drepState ->
                     drepState
-                      & drepExpiryL .~ addEpochInterval cgceCurrentEpoch ppDRepActivity
+                      & drepExpiryL
```

Comment:
This scenario didn't even occur, it is very nice that you thought of this.

Can you also check that we do the same thing where we refresh the DRep expiry on a vote submission.

I can't seem to find where we do that, which might be a bug if it is missing.

But the idea is that it is possible scenario:
* there to be no proposals when the epoch starts and the number of dormant epochs is `numDormantEpochs > 0`
* someone submits a proposal
* a DRep votes on that proposal, thus refreshing its expiry `curEpoch + ppDRepActivity`
* epoch boundary comes we incorrectly update the expiry for that DRep to be further in the future than it has to be: `curEpoch + ppDRepActivity + numDormantEpochs`

In other words similar problem as with update certificate, except it needs more conditions to occur.

Reply:
Awesome catch again :bow: